### ISSUE TITLE
NonceVerification: ignore unsetting of superglobal values

### ISSUE THREAD

**Issue by jrfnl** [WordPress/WordPress-Coding-Standards#1981]
## Bug Description

When all that's done with a superglobal value is to _unset_ it, no nonceverification should be necessary.

Currently, the sniff will throw a warning for this:
```
 148 | WARNING | Processing form data without nonce verification.
```

## Minimal Code Snippet

```php
// OK.
unset( $_GET['meta-box-loader'] );

// Not okay.
do_something_with( $_GET['meta-box-loader'] );
```

## Error Code
`WordPress.Security.NonceVerification.Recommended`

## Environment

| Question               | Answer
| ------------------------| -------
| PHP version             | any
| PHP_CodeSniffer version | `master`
| WPCS version            | `develop`
| WPCS install type       | N/A
| IDE (if relevant)       | N/A

## Additional Context (optional)
Might be a good idea to check in with the security team to confirm whether they agree with the above.

## Tested Against `develop` branch?
- [x] I have verified the issue still exists in the `develop` branch of WPCS.

**Comment by jrfnl** [comment]
Loosely related to #1902

**Comment by dd32** [comment]
FWIW I fully agree with this - Unsetting is not processing, and warning for lack of nonces when it's not processing it is just extra noise.
